flatpak/manifest_tool/flutter_ops.py

```
@@ -683,15 +927,29 @@ def _process_lotti_module(
+    # Replace sources but preserve ALL file sources for plugin dependencies
+    module["sources"] = (
+        [
+            {
+                "type": "archive",
+                "path": archive_name,
+                "sha256": sha256,
+                "strip-components": 1,
+            }
+        ]
+        + extra_sources
+        + existing_file_sources
```

Ruff (0.13.1)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)